### PR TITLE
sfeed: update to 1.2.

### PR DESCRIPTION
**Pull request by notthewave**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/removed-packages/template [hunk 1]
```diff
@@ -328,6 +328,7 @@ replaces="
  scantailor<=0.9.12.1_2
  seahorse-sharing<=3.8.0_4
  seriespl<=2.3.5_2
+ sfeed_curses<=1.0_1
```
Comment (Chocimier) [inline srcpkgs/removed-packages/template:331]: It should rather become transitional package to sfeed.

**Comment by notthewave** [comment]
@Chocimier is that how it's done?

**Comment by Chocimier** [comment]
Correct.
Conflicts an provides are better solved by transitional package. Not sure what make_use_env is for. vdoc readme look useful unless it repeats manpages.

**Comment by notthewave** [comment]
looks useful. tons of examples

**Comment by notthewave** [comment]
First I decied to remove it since some packages don't install base-sytem dependencies, but then I saw multiple listing it in depends= (including the original sfeed_curses). So I reverted the change again sorry for cloging up your inbox.

You can merge this. But maybe clarify if packages in base-system should be added to depends= or not (Manual.md).

**Comment by tibequadorian** [comment]
ncurses in depends= is not required because xbps already detects ncurses-libs as a required dependency.
Not because it's in base-system as users might also want to use base-minimal instead.

**Comment by notthewave** [comment]
Ok thanks for clarification.

**Comment by dkwo** [comment]
Why was `lynx` removed from `depends`?

**Comment by notthewave** [comment]
```
sfeed_content formats feed data (TSV) from sfeed(1) from stdin or for
     each file to stdout as plain-text content.  For HTML content it uses
     lynx(1) to convert it to plain-text.  At the end it uses the pager to
     view the output.  The sfeed_content script can be used by sfeed_curses(1)
     to view content.

SFEED_HTMLCONV
             The program used to convert HTML content to plain-text.  If it is
             not set it will use lynx by default.
```
It can be set by the user, however lynx is the default. 

Not sure what to prefer here.

**Pull request merged**